arangod/Cluster/Action.cpp

```diff
@@ -212,6 +212,11 @@ bool Action::operator<(Action const& other) const {
   return false;
 }
 
+void Action::addNewFactoryForTest(std::string const& name,
```

Comment:
I think this should be wrapped inside some `#ifdef ARANGODB_USE_GOOGLE_TESTS` ... `#endif` (same for the header file).
```suggestion
void Action::addNewFactoryForTest(std::string const& name,
```